### PR TITLE
Removed unneeded BOXNAME polling.

### PR DESCRIPTION
**Pull request by mikeller**
Currently `MSP_BOXNAME` is being polled 10 times a second. Taking into consideration that the response to this command is > 300 bytes, this is contributing the majority of the 'at rest' data transfer when a flight controller is connected:

Before:

image

After:

image

This behaviour was added about 5 years ago in [`0518ed6` (#19)](https://github.com/betaflight/betaflight-configurator/pull/19/commits/0518ed6993c0df30ed53e9f0ffd21bbb840acfcd), but it is not actually needed, as the list of supported modes only changes on startup.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]
Good catch!

Tested on MatekF411 using #10897 and this snippet in msp on line 345 to see what the hack is going on with all msp calls:
if (!requestExists) {
            function getKeyByValue(object, value) {
                return Object.keys(object).find(key => object[key] === value);
            }

            obj.timer = setInterval(function () {
                const directionStr = MSP.message_direction ? 'Out' : 'In';
                const codeStr = `${getKeyByValue(MSPCodes, code)} [${code}]`;
                GUI.log(`MSP data request timed-out: <strong>${codeStr}</strong> Direction: <strong>${directionStr}</strong> Tab: <strong>${GUI.active_tab}</strong> Callbacks: <strong>${MSP.callbacks.length}</strong>`);

                serial.send(bufferOut, false);
            }, 1000); // we should be able to define timeout in the future
        }

**Comment by hydra** [comment]
Please make sure you test on a board with and a board without a flash chip.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
